code/src/pages/MovieDetail.js

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect, useState } from 'react'
+import { useParams, Link, useHistory } from 'react-router-dom'
+import Company from '../pages/Companies'
+
+import BackButton from '../components/BackButton'
+const MovieDetail = () => {
+    const { id } = useParams()
+    const history = useHistory()
+    const [details, setDetails] = useState([])
+    const [statusCode, setStatusCode] = useState(200)
+    const [loading, setLoading] = useState(true)
+    const [prodComp, setProdComp] = useState([])
+
+    useEffect(() => {
+        fetch(`https://api.themoviedb.org/3/movie/${id}?api_key=2365aea36d60ef1f206bd1bdf23fd999&language=en-US`)
+        .then((res) => {
+            setStatusCode(res.status)
+            setLoading(false)
+            return res.json()  
+        }
+        )
+        .then((json) => {
+            setDetails(json)
+            setProdComp(json.production_companies)
+        })
+    },[])
+    useEffect(() => {
+        if (statusCode !== 200 || id.length > 19) {
+            history.push("/error")
+        }
+    }, [history, statusCode])
+
+    if (loading === true) {
+        return <div className="loading"><h1>loading...</h1></div>
+    }
+    return (
+        <div className='details-wrapper' 
+            style={{
+                backgroundImage: "url(https://image.tmdb.org/t/p/w1280" + details.backdrop_path
```

Comment:
This way of concatenating strings is used prior to ES6 - the new way is to use back tics🧹